webpack.config.js

```diff
@@ -1,8 +1,32 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require('path')
+
 module.exports = {
-  entry: './build/index.js',
+  target: 'web',
+  mode: 'production',
+  entry: './src/index.ts',
+  devtool: 'false',
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
```

Comment:
nit: We won't have any `.tsx` files in here

Reply:
Okay removed the optional x